src/citrine/resources/branch.py

```diff
@@ -61,6 +61,59 @@ def experiment_datasource(self) -> Optional[ExperimentDataSource]:
         branch_erds_iter = erds.list(branch_id=self.uid, version="latest")
         return next(branch_erds_iter, None)
 
+    def update_data(self,
```

Comment:
The SDK pattern is for making requests to happen in the corresponding collection class.

Reply:
The request was for this call to be on the branch itself, I updated it to use the existing methods on the collection to do the actual requesting part.  Will that work?

Reply:
I'll check with James. I'd like to preserve the existing pattern, but if he's dedicated to the method being on the `Branch` object itself, I'll give it the okay.